app/uk/gov/hmrc/rasapi/connectors/CachingConnector.scala

```diff
@@ -16,35 +16,37 @@
 
 package uk.gov.hmrc.rasapi.connectors
 
-import uk.gov.hmrc.rasapi.models.CustomerDetails
-
-trait CachingConnector {
-
-  def getCachedData(uuid: String): Option[CustomerDetails] = {
-
-    //TODO: Update this function to call our caching service(to be created) to access the data below
-
-    uuid match {
-      case "2800a7ab-fe20-42ca-98d7-c33f4133cfc2" => Some(CustomerDetails("LE241131B", "Jim", "Jimson", "1989-09-29"))
-      case "633e0ee7-315b-49e6-baed-d79c3dffe467" => Some(CustomerDetails("BB123456B", "John", "Smith", "1975-05-25"))
-      case "77648d82-309e-484d-a310-d0ffd2997791" => Some(CustomerDetails("LR325154D", "Jane", "Doe", "1969-06-09"))
-      case "79f21755-8cd4-4785-9c10-13253f7a8bb6" => Some(CustomerDetails("CC123456C", "Joe", "Bloggs", "1982-02-17"))
-      case "2900a7ab-fe20-42ca-98d8-c33f4133cfc2" => Some(CustomerDetails("PC243122B", "Peter", "Armstrong", "1969-01-01"))
-      case "743e0ee7-315b-49e7-baed-d79c3dffe467" => Some(CustomerDetails("EE123456D", "Steven", "Smith", "1947-08-15"))
-      case "88648d82-309e-484e-a310-d0ffd2997791" => Some(CustomerDetails("ZR132134C", "Simon", "Handyside", "1984-10-31"))
-      case "88648d82-309e-484d-a310-d0ffd2997792" => Some(CustomerDetails("SG123456D", "Linda", "Marshall", "1966-06-21"))
-      case "3000a7ab-fe20-42ca-98d9-c33f4133cfc2" => Some(CustomerDetails("CK355335C", "Kelly", "Thompson", "1990-02-15"))
-      case "853e0ee7-315b-49e8-baed-d79c3dffe467" => Some(CustomerDetails("AR355335C", "Simon", "Handyside", "1984-10-31"))
-      case "99648d82-309e-484f-a310-d0ffd2997791" => Some(CustomerDetails("NW424252D", "Zack", "Jackson", "1966-04-04"))
-      case "99648d82-309e-484d-a310-d0ffd2997793" => Some(CustomerDetails("KA122234B", "Linda", "Marshall", "1966-06-21"))
-      case "3100a7ab-fe20-42ca-98d1-c33f4133cfc2" => Some(CustomerDetails("WK332122D", "Oscar", "Smith", "1986-06-14"))
-      case "963e0ee7-315b-49e-baed-d79c3dffe467"  => Some(CustomerDetails("RW215443D", "Louise", "Oscar", "1966-04-04"))
-      case "11648d82-309e-484g-a310-d0ffd2997791" => Some(CustomerDetails("SE235112A", "Raj", "Patel", "1984-10-31"))
-      case "76648d82-309e-484d-a310-d0ffd2997794" => Some(CustomerDetails("AE325433D", "Mary", "Brown", "1982-02-17"))
-      case "76648d82-309e-484d-a310-d0ffd2997795" => Some(CustomerDetails("SG325433C", "Mary", "Poppins", "1952-02-17"))
-      case _ => None
+import play.api.http.Status.{FORBIDDEN, INTERNAL_SERVER_ERROR, NOT_ACCEPTABLE, NOT_FOUND}
+import uk.gov.hmrc.play.config.ServicesConfig
+import uk.gov.hmrc.play.http.{HeaderCarrier, HttpPost, Upstream4xxResponse, Upstream5xxResponse}
+import uk.gov.hmrc.rasapi.config.WSHttp
+import uk.gov.hmrc.rasapi.models.{CustomerDetails, Nino, ResidencyStatus}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+
+trait CachingConnector extends ServicesConfig {
+
+  val http: HttpPost = WSHttp
+  val cachingBaseUrl = baseUrl("caching")
```

Comment:
Better to leave unimplemented and add implementation to object.
If left as is, you will need to having an instance of the app in the unit tests so that the config values can be read.